server/controllers/lists.js

```diff
@@ -19,7 +19,15 @@ export async function explore(ctx, next) {
   const contentPromos = contentList.map(PromoFactory.fromArticleStub);
   const promos = wpPromos.concat(contentPromos).sort((a, b) => {
     return a.datePublished.getTime() - b.datePublished.getTime();
-  }).reverse();
+  })
+  .reverse()
+  .map((promo, index) => {
+    if (index === 0) {
+      return Object.assign({}, promo, {weight: 'lead'});
+    } else {
+      return promo;
+    }
+  });
```

Comment:
Could you leave a note here saying what this is doing so we remember to add it in properly one 🌧  day.